lib/sites-base.js

```diff
@@ -278,15 +282,16 @@ module.exports = {
         }
 
         async function copyDatabase() {
+          // The copyDb command is gone in mongodb >= 4.2, use
+          // mongodump and mongorestore with --uri and --archive
+          // (works in 3.4.something and up)
           const shortNamePrefix = self.apos.shortName.replace(/dashboard$/, '');
-          // http://stackoverflow.com/questions/36403749/how-can-i-execute-db-copydatabase-through-nodejss-mongodb-native-driver
-          const command = {
-            copydb: 1,
-            fromdb: shortNamePrefix + piece.copyOfId,
-            todb: shortNamePrefix + piece._id
-          };
-          const admin = self.apos.db.admin();
-          return admin.command(command);
+          const baseUri = self.apos.options.multisite.mongodbUrl;
+          const fromDb = `${shortNamePrefix}${piece.copyOfId}`;
+          const toDb = `${shortNamePrefix}${piece._id}`;
+          const fromUri = `${baseUri}/${fromDb}`;
```

Comment:
An issue I mentioned before: `fromUri` might already have a database name in it (definitely will in production). You will need to parse it, set a new `path`, and convert it back to a string.

Reply:
`baseUri` might, rather, thus the problem assembling `fromUri` and `toUri` in this naive way like I was able to get away with for the demo site.